apps/frontend/src/app/poll/choose-events/choose-events.component.ts

```diff
@@ -202,13 +210,32 @@ export class ChooseEventsComponent implements OnInit {
     } else if (maxParticipants && this.participants.length >= maxParticipants) {
       this.closedReason = 'This poll has reached it\'s maximum number of participants. You can no longer submit your vote.';
       this.showResults = true;
```

Comment:
This seems wrong

Reply:
It was there before, but yeah you are right. Now that in case of NEVER the number of `this.participants` has changed, we might refactor to let the backend make this check instead.

Reply:
I believe it is no longer necessary to set this here, as it should be handled by updateHiddenReason